extensions/2.0/Khronos/KHR_texture_astc/schema/texture.KHR_texture_basisu.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_texture_astc glTF Texture Extension",
+    "type": "object",
+    "description": "glTF extension to specify textures using the KTX v2 images with astc compression.",
```

Comment:
nit: astc => ASTC

Reply:
Fixed in https://github.com/KhronosGroup/glTF/pull/2369/commits/94e429323ad4c901cc31a2d9deb56e18d718fe21